### ISSUE TITLE
Edge labels              & dataset.py

### ISSUE THREAD

**Issue by 48-1-del** [suy9/ESCNet#6]
Author, in your dataset.py file, you wrote code requiring the dataset to contain a GT_Edge folder and that the edge label file for each image must exist.

Otherwise, it will throw an exception and fail to run.

Edge labels are used for preprocessing and training during training. So where did you get the edge images for this training set?

if self.is_train:
    # Get ground truth edges
    self.edge_paths = []
    for p in self.image_paths:
        base, ext = os.path.splitext(p)  # 分离文件名和扩展名
        p_gt = os.path.join(
            os.path.dirname(p).replace("Image", "GT_Edge"),
            os.path.basename(base) + ".png",
        )
        if os.path.exists(p_gt):
            self.edge_paths.append(p_gt)

    if len(self.edge_paths) != len(self.image_paths):
        set_image_paths = set(
            [os.path.splitext(p.split(os.sep)[-1])[0] for p in self.image_paths]
        )
        set_edge_paths = set(
            [os.path.splitext(p.split(os.sep)[-1])[0] for p in self.edge_paths]
        )
        print("Path diff:", set_image_paths - set_edge_paths)
        raise ValueError(
            f"There are different numbers of images ({len(self.edge_paths)}) and edges ({len(self.image_paths)})"
        )

**Comment by suy9** [comment]
Edge images are included in each dataset. If you don't have this part, it may be because you didn't download the complete dataset.
